python313Packages.pymodbus: 3.11.1 -> 3.11.2

fabaff opened the pull request:
Diff: https://github.com/pymodbus-dev/pymodbus/compare/v3.11.1...v3.11.2

Changelog: https://github.com/pymodbus-dev/pymodbus/releases/tag/v3.11.2


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python313Packages.pymodbus: 3.11.1 -> 3.11.2

Diff: https://github.com/pymodbus-dev/pymodbus/compare/v3.11.1...v3.11.2

Changelog: https://github.com/pymodbus-dev/pymodbus/releases/tag/v3.11.2

sikmir commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 442189`
Commit: [`9ddd3cb2a966ef2b1a378ccbfe0c6f3aaeb8d46b`](https://github.com/NixOS/nixpkgs/commit/9ddd3cb2a966ef2b1a378ccbfe0c6f3aaeb8d46b) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/9ddd3cb2a966ef2b1a378ccbfe0c6f3aaeb8d46b..pull/442189/head))
Merge: [`0f0c9e2bf07b5b8e093e58de468e8b5758bbb661`](https://github.com/NixOS/nixpkgs/commit/0f0c9e2bf07b5b8e093e58de468e8b5758bbb661)

Logs: https://github.com/sikmir/nixpkgs-review-gha/actions/runs/18050170324


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 27 packages built:</summary>
  <ul>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>home-assistant-component-tests.iskra</li>
    <li>home-assistant-component-tests.modbus</li>
    <li>home-assistant-component-tests.smarty</li>
    <li>home-assistant-component-tests.stiebel_eltron</li>
    <li>home-assistant-custom-components.solax_modbus</li>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 27 packages built:</summary>
  <ul>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>home-assistant-component-tests.iskra</li>
    <li>home-assistant-component-tests.modbus</li>
    <li>home-assistant-component-tests.smarty</li>
    <li>home-assistant-component-tests.stiebel_eltron</li>
    <li>home-assistant-custom-components.solax_modbus</li>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python312Packages.pymodbus</summary>
<pre>FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TCP-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TLS-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.UDP-localhost] - assert False
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_receive_data - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_server_trace_connect_disconnect - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_connection_lost - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_shutdown_connection - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_no_device - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_modbus_error - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_tls_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tls_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_udp_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_close - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_exception - RuntimeError: Could not start listen, please check address.
====== 23 failed, 1426 passed, 6 warnings, 16 errors in 69.68s (0:01:09) =======</pre>
</details>
<details>
<summary>python313Packages.pymodbus</summary>
<pre>FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TCP-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TLS-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.UDP-localhost] - assert False
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_receive_data - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_server_trace_connect_disconnect - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_connection_lost - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_shutdown_connection - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_no_device - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_modbus_error - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_tls_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tls_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_udp_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_close - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_exception - RuntimeError: Could not start listen, please check address.
====== 23 failed, 1426 passed, 5 warnings, 16 errors in 69.57s (0:01:09) =======</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.pymodbus</summary>
<pre>FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TCP-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TLS-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.UDP-localhost] - assert False
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_receive_data - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_server_trace_connect_disconnect - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_connection_lost - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_shutdown_connection - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_no_device - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_modbus_error - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_tls_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tls_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_udp_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_close - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_exception - RuntimeError: Could not start listen, please check address.
====== 23 failed, 1426 passed, 5 warnings, 16 errors in 62.70s (0:01:02) =======</pre>
</details>
</details>

sikmir commented:
Regression on darwin.

Zaczero commented:
Failed to start server [Errno 1] error while attempting to bind on address ('127.0.0.1', 0): [errno 1] operation not permitted

@sikmir Not a permissions/sandboxing issue?

fabaff committed:
python313Packages.pyiskra: 0.1.26 -> 0.1.27

Diff: https://github.com/Iskramis/pyiskra/compare/v0.1.26...v0.1.27

Changelog: https://github.com/Iskramis/pyiskra/releases/tag/v0.1.27
